### ISSUE TITLE
$(...).tooltip is not a function

### ISSUE THREAD

**Issue by tubsandcans** [shakacode/bootstrap-loader#245]
I'm trying to replace sprockets with webpack in my rails app which uses bootstrap 3. So far I'm able to use webpack to serve assets from node installations of jquery, knockout, and bootstrap-loader. 

When I hover over an element with a .tooltip binding, I get the '...is not a function' error. It appears that bootstrap styles and javascripts are being included and loaded (the only other component I'm having issues with are modals, which show up for about a split second before disappearing). I can even type $('#element_id').tooltip in js console and it returns the tooltip function(!). How can this be?

**Comment by justin808** [comment]
@tubsandcans Please try this out on a fork of https://github.com/shakacode/react-webpack-rails-tutorial and let's solve this together!

**Comment by tubsandcans** [comment]
TL;DR I'm going to try using the react-rails gem going forward. 

Initially I just wanted to replace sprockets with webpack as a way to understanding the ins and outs of webpack. This being in service to my eventual goal of getting react integrated into my current rails deployment. I'm now realizing how much of a complete PITA this is. I followed the instructions in [this post](http://pixelatedworks.com/articles/replacing-the-rails-asset-pipeline-with-webpack-and-yarn/) and everything seemed pretty straightforward. That is until I ran into issues with bootstrap-loader and Datatables being bundled but throwing errors on page load. I have never seen anything quite like this, its scary.

**Comment by justin808** [comment]
@tubsandcans You're throwing out the baby with the bathwater...

Instead of replacing the asset pipeline, just use the asset pipeline to include the necessary assets using the https://github.com/shakacode/react_on_rails technique. 

You're going backwards if you're going to the react-rails gem and only using the asset-pipeline rather than Webpack.

If you want this to go super fast, [ShakaCode](http://www.shakacode.com) is available to help you. Get in touch with me directly at [justin@shakacode.com](mailto:justin@shakacode.com).

![2017-01-30_11-33-59](https://cloud.githubusercontent.com/assets/1118459/22536281/a592cd8e-e8a4-11e6-9b7d-22eeb7036817.png)
